opba-auth-rest-impl/src/main/java/de/adorsys/opba/tppauthapi/controller/PsuAuthController.java

```diff
@@ -10,7 +10,7 @@
 import de.adorsys.opba.protocol.facade.services.authorization.PsuLoginForAisService;
 import de.adorsys.opba.protocol.facade.services.psu.PsuAuthService;
 import de.adorsys.opba.api.security.internal.config.AuthorizationSessionKeyConfig;
-import de.adorsys.opba.tppauthapi.config.CookieProperties;
+import de.adorsys.opba.api.security.internal.config.CookieProperties;
```

Comment:
Although it is not your fault, but we need to add COOKIE_TTL header to `loginForApproval` and `login`

Reply:
done